sql/table_test.go

```diff
@@ -168,7 +175,9 @@ func TestMakeTableDescIndexes(t *testing.T) {
 		}
 		schema, err := makeTableDesc(stmt[0].(*parser.CreateTable))
 		if err != nil {
```

Comment:
I don't think this test is being polluted. Having a separate test is an overkill. It's testing all the functionality and is easy to understand.